docs/modules/ROOT/pages/how-tos/appcat/qppuio-quotas.adoc

```diff
@@ -0,0 +1,35 @@
+= Change AppCat service quota on APPUiO Cloud
+
+It's important, that the service instance has been created with resources lower than the quota.
+This is due to the fact, that the quotas are bound to the namespace and in order to change them, we need the namespace itself.
+
+== Namespace quotas
+The amount of namespaces and thus the amount of AppCat instances that can be created for an organization on APPUiO is by default 25.
+
+If that limit is reached, the customer can request an increase in the amount of namespaces they can have.
+This is controlled by a ConfigMap with the name `overrides-$orgName` in the namespace `appuio-cloud`.
```

Comment:
Wouldn't it make more sense to point to the APPUiO Cloud documentation, rather than document APPUiO Cloud specifics here?